app/views/rewards/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Reward page</h1>
```

Comment:
Do you need a show page for one reward? Wouldn't an admin just want to see all the rewards with their cost on the index and thats it? Just trying to think about it given a real world context.